Makefile

```diff
@@ -478,9 +478,9 @@ kubectl config use-context $1 > /dev/null
 endef
 
 define hit-testapp-host
-	kubectl run -it --rm busybox --restart=Never --image=busybox -- sh -c \
-	"echo 'nameserver `$(K8GB_COREDNS_IP)`' > /etc/resolv.conf && \
-	wget -qO - $1"
+	kubectl run -it --rm busybox --restart=Never --image=busybox --command \
+	--overrides='{"spec": {"dnsConfig": {"nameservers": ["$(K8GB_COREDNS_IP)"]}, \"dnsPolicy\": \"None\"}}' \
+	-- wget $1
```

Comment:
`-qO -` is missing (we can use curl actually instead) (`-q` ... quiet (no wget stuff), `-O -` print to stdout)

Reply:
@jkremser good catch! Amended